sdcm/nemesis.py

```diff
@@ -5076,21 +5079,25 @@ def disrupt_bootstrap_streaming_error(self):
 
         bootstrapabortmanager = NodeBootstrapAbortManager(bootstrap_node=new_node, verification_node=self.target_node)
 
-        bootstrapabortmanager.run_bootstrap_and_abort_with_action(terminate_pattern, abort_action=new_node.stop_scylla)
-        bootstrapabortmanager.clean_and_restart_bootstrap_after_abort()
+        with ignore_stream_mutation_fragments_errors(), ignore_raft_topology_cmd_failing(), ignore_raft_transport_failing():
+            bootstrapabortmanager.run_bootstrap_and_abort_with_action(
+                terminate_pattern, abort_action=new_node.stop_scylla)
+            bootstrapabortmanager.clean_and_restart_bootstrap_after_abort()
 
-        if new_node.db_up() and not self.target_node.raft.is_cluster_topology_consistent():
-            LOGGER.error("New host %s was not properly bootstrapped. Terminate it", new_node.name)
-            bootstrapabortmanager.clean_unbootstrapped_node()
-            self.log.info("Failed bootstrapped node %s removed. Cluster is in initial state", new_node.name)
-            raise BootstrapStreamErrorFailure(f"Node {new_node.name} failed to bootstrap")
+            if new_node.db_up() and not self.target_node.raft.is_cluster_topology_consistent():
```

Comment:
in case of `if new_node.db_up()` returns `False` we'll be left with dangling node till the end of the test.

Reply:
This case is processed by bootstrapabortmanger.clean_and_restart_bootstrap_after_abort method. It checks that scylla is up or raise an exception if it failed

Reply:
ok, so in that case, why bothering about `new_node.db_up()` then?

Reply:
ok, now I see,
But now when node is removed, new_node.db_up will always show false and proceed with decomission.
So in that case, first we need to check if node still exists in the cluster I think.